runtime/default.go

```diff
@@ -195,6 +195,12 @@ func (r *runtime) Create(s *Service, opts ...CreateOption) error {
 		o(&options)
 	}
 
+	dir, err := filepath.Abs(filepath.Dir(os.Args[0]))
```

Comment:
os.Args feels heavy handed when we have proper cli args etc in outer layers. Suggestions welcome.